src/provideInlineCompletionItems.ts

```diff
@@ -80,17 +81,12 @@ function calculateRange(
   );
 }
 
-function isInTheMiddleOfWord(
+function isValidMidlinePosition(
   document: vscode.TextDocument,
   position: vscode.Position
 ): boolean {
-  const nextCharacter = document.getText(
-    new vscode.Range(position, position.translate(0, 1))
+  const lineSuffix = document.getText(
+    new vscode.Range(position, position.with({ character: 500 }))
```

Comment:
use `document.lineAt(position.line).range.end` instead of `position.with({ character: 500 })`